elements/pfe-autocomplete/src/pfe-autocomplete.js

```diff
@@ -309,14 +309,23 @@ class PfeAutocomplete extends PFElement {
     this._closeDroplist();
   }
 
+  /**
+   * Returns the HTML of the active element
+   * @param {integer} activeIndex Index of an element in the droplist
+   * @return The HTML inside of the given index
```

Comment:
```suggestion
   * @return {string} The HTML inside of the given index as a string
```

I updated the return note here but (and I know this is out of scope of the work you're doing), but I might suggest returning the element itself for microfunctions like this in the future - it's more reusable.

Reply:
@castastrophe ahh gotcha, that makes sense. I really appreciate the feedback!

Reply:
Thanks for adding all this awesome documentation 🎉